frontend/js/components/LocationField.vue

```diff
@@ -172,6 +172,11 @@
 
         this.address = newValue
         this.$emit('change', newValue)
+
+        const latlng = newValue.match(/^(-?\d+(?:\.\d+)?),+ *(-?\d+(?:\.\d+)?)$/)
+        if (latlng) {
+          this.onLatLngEntered(latlng[1], latlng[2]);
```

Comment:
```suggestion
          this.onLatLngEntered(latlng[1], latlng[2])
```

Small fix for eslint